src/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -196,6 +196,9 @@ internal static void RhWaitForPendingFinalizers(bool allowReentrantWait)
         [RuntimeImport(RuntimeLibrary, "RhGetAllocatedBytesForCurrentThread")]
         internal static extern long RhGetAllocatedBytesForCurrentThread();
 
+        [DllImport(RuntimeLibrary, ExactSpelling = true, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern long RhGetTotalAllocatedBytes(bool precise);
```

Comment:
What is the difference between `Rh` and `Rhp` prefix?

Reply:
> The difference between Rh and Rhp prefix: 

https://github.com/dotnet/corert/issues/3294#issuecomment-330615488 

> RhWaitForPendingFinalizers e.g. does not specify cdecl in DllImport but does inside the C++ code.

This is a bug. The calling conventions should match, but they matter on x86 only that does not work in CoreRT today. 

Ideally, we would just use the default everywhere, without explicitly specifying the calling convention anywhere (it is the plan that CoreCLR is on). We were not able to do it originally because of ProjectN quirks. 

It should be ok to not specify the calling convention for any new code. We may do a pass a strip them on the existing code.